webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -238,6 +242,23 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
             </>
           </>
         )}
+        {!canSetWorkspaceNamespaceBindings &&
+          canViewResourceQuotas &&
+          isEditing &&
+          info.branding === BrandingType.HPE &&
+          resourceManagers.length > 0 && (
+            <>
+              <Divider />
+              Resource Quotas
+              <>
+                {resourceManagers.map((name) => (
+                  <Fragment key={name}>
+                    <Form.Item label={`Cluster Name: ${name}`} name={['resourceQuotas', name]} />
+                  </Fragment>
```

Comment:
nit: this doesn't need to be wrapped in a fragment
```suggestion
                  <Form.Item key={name} label={`Cluster Name: ${name}`} name={['resourceQuotas', name]} />
```

Reply:
If I don't wrap it in that i get this err:

```
/Users/salonigupta/determined/webui/react/src/components/WorkspaceCreateModal.tsx
  255:19  error  Missing "key" prop for element in iterator  react/jsx-key

✖ 1 problem (1 error, 0 warnings)
```

Reply:
you just need the `key` prop added to the `Form.Item` component

Reply:
gotcha! fixed it